projects/1.16/assets/fluid-system/fluidsystem/lang/zh_cn.json

```
@@ -0,0 +1,30 @@
+{
+    "block.fluidsystem.controller_tank_block": "储罐方块控制器",
+    "block.fluidsystem.infinite_water_source": "无限水源",
+    "block.fluidsystem.input_pipe": "输入管道",
+    "block.fluidsystem.input_tank_block": "储罐方块输入",
+    "block.fluidsystem.interface_tank_block": "储罐方块接口",
+    "block.fluidsystem.output_pipe": "输出管道",
+    "block.fluidsystem.pipe": "管道",
+    "block.fluidsystem.pipe_controller": "管道控制器",
+    "block.fluidsystem.structural_tank_block": "储罐方块",
+    "gui.fluidsystem.liquidAmount": "%smB/%smB",
+    "gui.fluidsystem.liquidPercentage": "流体： %s",
+    "gui.fluidsystem.tanksBlock": "储罐的方块： ",
+    "itemGroup.fluidsystem_tab": "流体系统",
+    "message.fluidsystem.fluidFilterRemoved": "移除流体过滤器",
+    "message.fluidsystem.fluidFiltered": "流体过滤： %s",
+    "message.fluidsystem.fluidFilteredSet": "流体过滤器设置： %s",
+    "message.fluidsystem.tankControllerMissing": "结构内缺少控制器",
+    "message.fluidsystem.tankExtraController": "结构内含有多个控制器",
+    "message.fluidsystem.tankFormed": "结构建造正确",
+    "message.fluidsystem.tankTooBig": "结构大小超过上限",
+    "messagefluidsystem.tankEmpty": "空",
+    "tooltip.fluidsystem.inputPipe": "你可以从管道中输入流体，同时发出红石信号",
+    "tooltip.fluidsystem.outputPipe": "允许从管道中输出流体",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

表述很奇怪。一般来说，允许的后面得加上人称代词，比如允许xx做xx，。但是在中文语境里，允许一根管道输出液体，从语义角度出发就已经很奇怪了。管道不需要别人允许，它自己就能工作，允许管道这一表述完全说不通。

看了看原文，你的翻译不仅表述有问题，而且和原文也有较大出入。建议做如下修改：
```suggestion
    "tooltip.fluidsystem.outputPipe": "该方块能够让你使用管道排出流体",
```
表述还是很奇怪，但是原文就很奇怪，所以一起奇怪吧。
顺便上面那句也得改。